gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/metastore/HiveMetaStoreBasedRegister.java

```diff
@@ -93,6 +93,8 @@
 
   public static final String HIVE_REGISTER_METRICS_PREFIX = "hiveRegister.";
   public static final String ADD_PARTITION_TIMER = HIVE_REGISTER_METRICS_PREFIX + "addPartitionTimerTimer";
+  public static final String SCHEMA_SOURCE_DB = HIVE_REGISTER_METRICS_PREFIX + "schema.source.dbName";
+  public static final String SCHEMA_SOURCE_TABLE = HIVE_REGISTER_METRICS_PREFIX + "schema.source.tableName";
```

Comment:
I would rather we getting rid of source table here for simplicity (and add javadoc indicating our assumption that the actual source table will be named the same with only database name being different). wdyt ?

Reply:
Will address this.